### ISSUE TITLE
Suggestion: Update wiki and close TranslationZed-related issues

### ISSUE THREAD

**Issue by heldinz** [TheIndieStone/ProjectZomboidTranslations#997]
As a PZ translation n00b, I've just read a bunch of documentation scattered across different sites to try to understand how the system and process works. As far as I can tell, this repo and the [How to Translate Project Zomboid](https://steamcommunity.com/sharedfiles/filedetails/?id=3006690572) guide over on Steam are the most up-to-date resources, and according to comments on that guide, TranslationZed is outdated.

If that is all correct, I'd like to make the following suggestions.

1. Remove basically everything that's currently in the wiki, specifically:
  - The TranslationZed User Guide
  - The links to the outdated forum posts
2. Add a link from the wiki to the Steam guide instead
3. Close the following issues, as they are all years-old and reporting bugs with TranslationZed:
  - #171 
  - #617 
  - #703

**Comment by Faalagorn** [comment]
I guess it's not a terrible idea, the GitHub wiki is from the time Connall used to manage it, but he's no longer managing it and busy IRL, so it's just Nasko managing the GitHub, though all the translation updates are frozen until build 42 – not sure if any further fixes are planned after build 42 hits as far as translation goes, guess we'll see.

The Steam guide is indeed the most popular and accurate resource, it's also pinned on the Discord. It might also not be a terrible idea to basically move some of the content to the PZwiki so people can edit it there too. We have a newly created page for translations https://pzwiki.net/wiki/Translations that moved the table from modding https://pzwiki.net/wiki/Modding and we https://pzwiki.net/wiki/Template:Navbox_modding where we plan to eventually have pages outline the tools there, including TranslationZed.

**Comment by heldinz** [comment]
Makes sense, I think the wiki is probably the first place most people start looking for info in general 👍🏻
